interapp-backend/api/routes/middleware.ts

```diff
@@ -23,7 +23,7 @@ export function validateRequiredFields<T extends z.ZodType<ReqBody | ReqQuery>>(
       );
     }
     if (req.method === 'GET') {
-      req.query = validationResult.data as ReqQuery;
+      res.locals.query = validationResult.data as ReqQuery;
     } else {
       req.body = validationResult.data as ReqBody;
```

Comment:
can u make req.body => res.locals.body too? 
it'll probably be better code design to not touch the raw req.body in the middleware, my mistake on that

Reply:
Done :D